remove non alpha numeric chars in env name as model param

jbbonifacio opened the pull request:
remove non alpha numeric characters in env name when used as model parameter to avoid having issues

example:
<img width="1072" height="281" alt="image" src="https://github.com/user-attachments/assets/18595a57-8df9-4c72-b0a4-b0d9ec0a7cd2" />

loregbatista commented:
I'm not sure the changes you've made propagate well throughout Bioflow. If you drop the alphanumeric characters in some modules and not the others, this could lead to incompatibility issues in the data.
This either needs to be self-contained within the modules or implemented from the beginning.
Options are:

1) Store the original names and revert back to them once the analysis is done
2) Remove the alphanumeric characters as soon as data is loaded

jbbonifacio commented:
Additional updates:
**app_ui**
- hide lme4 and sommer from the available engines in Multi-Trial Analysis module

**mod_mtaLMMsolveApp**
- updated the UI
1. moved the surrogates of merit in line with the shortcut to popular genetic evaluation models
NOTE: I noticed that when we update the choices of the radio buttons, the tooltip is not showing, so I commented it out first. maybe someone can help or knows a way to fix it
2. added an info for Marker QA version to use

**mod_saveData**
- added a summary table to show a summary of the data available (pheno, geno, pedigree, QTL, weather) in the current object

**reportMtaLMMsolver**
- removed non alpha numeric characters for model parameters

**reportSta**
- fix issue encountered in field view plot (heatmap) when 1st occurrence do not have available row and column information

jbbonifacio commented:
Hello @loregbatista and @khaled-alshamaa, I already changed the computation of environment column as suggested. I removed the non alpha numeric characters from the columns forming the environment column and then concatenated them with _ as separator. I also added "env" as prefix. I just copied the way it was updated before when used as parameter in the model.

example:
year = 2024
season = WS
location = IRRI-HQ

environment = env2024_WS_IRRIHQ

Thank you!
